### PR TITLE
Translate Ruby 2.7.0-preview3 released news (id)

### PR DESCRIPTION
**Pull request by meisyal**
cc: @kuntoaji

### CODE REVIEW

File: id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+translator: "meisyal"
+date: 2019-11-23 12:00:00 +0000
+lang: id
+---
+
+Kami dengan senang hati mengumumkan rilis dari Ruby 2.7.0-preview3.
+
+Sebuah versi *preview* dirilis untuk mengumpulkan umpan balik rilis final
+yang direncanakan rilis pada bulan Desember. Versi ini dirilis terutama untuk
+mengkonfirmasi *compatibility* dari *keyword argument*.
```
Comment (kuntoaji) [inline id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md]: instead of compatibility, how about "kompatibilitas"?

Comment (meisyal) [reply id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md]: I thought "kompatibilitas" is not a valid word in Bahasa. It's available on KBBI https://kbbi.web.id/kompatibilitas.

OK. I will translate all the "compatibility" to "kompatibilitas".

Comment (meisyal) [reply id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md]: I thought "kompatibilitas" was not a valid word in Bahasa. It's available on KBBI https://kbbi.web.id/kompatibilitas.

OK. I will translate all the "compatibility" to "kompatibilitas".

Comment (meisyal) [reply id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md]: Translated to "kompatibilitas" @kuntoaji

File: id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    murni.
+
+  * Nilai default dari `--jit-min-calls` diubah dari 5 ke 10,000.
+
+  * Nilai default dari `--jit-max-cache` diubah dari 1,000 ke 100.
+
+* ~~`Symbol#to_s`, `Module#name`, `true.to_s`, `false.to_s`, dan `nil.to_s`
+  sekarang selalu mengembalikan sebuah *frozen string*. *String* yang
+  dikembalikan selalu sama untuk sebuah objek. [Eksperimental]
+  [[Feature #16150]](https://bugs.ruby-lang.org/issues/16150)~~ dikembalikan
```
[the remark this answers is not in the document]
Comment (meisyal) [reply id/news/_posts/2019-11-23-ruby-2-7-0-preview3-released.md]: Ah, thank you for pointing this out. I have corrected that @aleksandrilyin

**Pull request merged**